react/features/virtual-background/components/VirtualBackgroundPreview.tsx

```diff
@@ -1,29 +1,40 @@
-// @flow
-
+/* eslint-disable lines-around-comment */
 import Spinner from '@atlaskit/spinner';
 import { withStyles } from '@material-ui/core/styles';
 import React, { PureComponent } from 'react';
+import { WithTranslation } from 'react-i18next';
 
+import { IState } from '../../app/types';
+// @ts-ignore
 import { hideDialog } from '../../base/dialog';
+// @ts-ignore
 import { translate } from '../../base/i18n';
-import { VIDEO_TYPE } from '../../base/media';
+// @ts-ignore
 import Video from '../../base/media/components/Video';
-import { connect, equals } from '../../base/redux';
+import { VIDEO_TYPE } from '../../base/media/constants';
+import { connect, equals } from '../../base/redux/functions';
+// @ts-ignore
 import { getCurrentCameraDeviceId } from '../../base/settings';
+// @ts-ignore
 import { createLocalTracksF } from '../../base/tracks/functions';
+// @ts-ignore
 import { NOTIFICATION_TIMEOUT_TYPE } from '../../notifications';
+// @ts-ignore
 import { showWarningNotification } from '../../notifications/actions';
+// @ts-ignore
 import { toggleBackgroundEffect } from '../actions';
 import { VIRTUAL_BACKGROUND_TYPE } from '../constants';
+// @ts-ignore
 import { localTrackStopped } from '../functions';
+// @ts-ignore
 import logger from '../logger';
 
 const videoClassName = 'video-preview-video';
 
 /**
  * The type of the React {@code PureComponent} props of {@link VirtualBackgroundPreview}.
  */
-export type Props = {
+export type Props = WithTranslation & {
```

Comment:
type with & vs interface with extends. We should pick 1 and stick to it.

Reply:
In general I prefer interface with extends. But here I left it as type because it's exported, which means some Flow file imports it and I'm not sure if changing it to interface would break it or not

Reply:
Shouldn't, since all Flow types are stripped by the babel plugin.